docs/sphinx/compiling/dpdk.rst

```diff
@@ -85,9 +78,9 @@ Add the following to ``/etc/rc.local``, and make sure ``rc.local`` is executable
     echo 1024 > /sys/kernel/mm/hugepages/hugepages-2048kB/nr_hugepages
 
     modprobe uio
-    insmod /opt/dpdk-stable-16.11.10/x86_64-native-linuxapp-gcc/kmod/igb_uio.ko
-    PCI_NIC_LIST=`/opt/dpdk-stable-16.11.10/tools/dpdk-devbind.py --status | grep X710 | cut -b 6-12 | tr '\n' ' '`
-    /opt/dpdk-stable-16.11.10/tools/dpdk-devbind.py --bind igb_uio $PCI_NIC_LIST
+    insmod /opt/dpdk/x86_64-native-linuxapp-gcc/kmod/igb_uio.ko
```

Comment:
I need to update these DPDK docs...  This is correct for the existing CHIME install, but the procedure is different now...